src/CommunityPatch/Patches/EarlyStoryVisibleTimeoutPatch.cs

```diff
@@ -54,18 +54,23 @@ public sealed class EarlyStoryVisibleTimeoutPatch : IPatch {
       yield break;
     }
 
-    private static int ExtractFirstPhaseTimeLimitInYears() {
+    private static int? ExtractFirstPhaseTimeLimitInYears() {
       if (FirstPhaseTimeLimitInYearsField == null)
-        return 0;
+        return null;
 
       try {
         return (int) FirstPhaseTimeLimitInYearsField
           .GetRawConstantValue();
       }
-      catch {
-        return (int) FirstPhaseTimeLimitInYearsField
-          .GetValue(FirstPhaseTimeLimitInYearsField.IsStatic ? null : FirstPhaseInstance);
+      catch {}
+      try {
+        if (FirstPhaseTimeLimitInYearsField.IsStatic) {
+          return (int) FirstPhaseTimeLimitInYearsField.GetValue(null);
+        }
       }
+      catch {}
+
+      return null;
```

Comment:
To avoid throwing if we can't retrieve the value as either a constant or a static and just return null.

Reply:
Reverted this part.